_client/src/Dialogs/DialogManager.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <QString>
+#include <QObject>
+#include <QMessageBox>
+
+#include "LoginDialog.hpp"
+#include "RegistrationDialog.hpp"
+#include "Network/ApiManager.hpp"
+
+/**
+ * @class DialogManager
+ * Controls all dialogs in MessageBoxes
+ */
+class DialogManager : public QObject
```

Comment:
make it final https://en.cppreference.com/w/cpp/language/final

Reply:
done